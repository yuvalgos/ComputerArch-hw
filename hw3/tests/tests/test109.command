./dflow_calc tests/test109.dat tests/test109.in d13 d107 p10 d106 d93 d171 p52 p25 p169 p171 p163 d199 p67 p10 d52 p40 p120 p18 d102 p95 d45 d178 d106 p139 d178 p4 d50 d40 d110 d133 d61 p55 p119 d104 d27 p11 p43 d36 d125 d172 d198 p30 d133 p195 p148 d82 d89 p27 p83 p75 d30 p186 p155 p0 d167 p32 p170 p74 p64 p17 d117 p104 d82 d186 p10 d92 p123 d96 p144 d96 d146 d142 p123 d7 d171 p192 d95 d187 d177 p152 p171 d137 p143 p58 p106 p16 p87 p156 p48 p43 p11 p9 d51 p70 p192 d33 p75 p74 d199 d73 d162 p47 p149 d13 p21 d78 p195 d0 p132 p30 p106 p45 p139 p22 d95 p99 d160 d75 p123 p106 d168 d18 p80 p28 p120 p64 p30 d159 p119 d98 d180 p183 p11 d55 p132 d165 p55 d64 p193