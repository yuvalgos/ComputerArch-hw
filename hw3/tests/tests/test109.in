8 24 25 17
1 10 10 4
8 14 2 21
5 26 12 0
7 22 3 17
5 31 29 1
3 24 17 19
2 12 17 18
1 30 23 18
4 21 27 1
7 17 23 31
3 22 28 13
5 1 17 9
7 29 4 3
2 25 12 15
5 17 30 29
5 5 21 20
2 24 17 1
8 22 11 20
2 13 8 2
6 6 21 4
7 14 6 17
3 27 25 9
3 9 6 1
5 10 12 20
8 30 1 23
0 29 26 29
2 26 0 19
2 17 24 24
4 14 5 17
5 25 2 4
6 15 22 14
2 16 16 28
0 4 11 23
4 22 26 9
0 19 5 8
2 18 22 24
4 9 23 23
2 26 9 18
6 29 21 25
4 14 27 17
0 17 6 7
7 13 17 7
1 18 26 3
0 31 12 25
8 17 9 20
8 25 6 2
6 26 17 28
4 29 10 1
7 25 18 28
4 28 8 13
1 5 30 5
7 21 14 31
8 29 26 24
0 16 13 25
8 2 4 5
8 22 7 1
4 9 3 7
5 12 9 29
1 21 15 1
2 19 15 3
0 26 17 13
2 7 30 22
8 3 12 29
5 29 2 25
1 9 16 3
2 13 12 30
4 15 27 7
0 1 18 28
0 9 0 23
7 29 17 27
6 5 8 13
5 15 3 11
0 21 6 23
5 8 16 11
2 23 27 30
3 19 9 4
8 25 11 9
1 28 17 26
2 24 13 25
6 17 22 10
8 24 30 18
0 20 10 21
7 8 27 26
7 9 7 8
5 11 25 4
1 9 12 6
1 3 10 10
8 10 0 13
1 15 16 31
7 13 19 3
5 30 8 19
1 23 7 18
6 21 14 4
1 21 5 22
1 30 0 21
8 11 22 2
3 12 31 0
0 4 16 25
3 27 31 28
5 19 6 13
0 22 29 14
4 0 8 10
7 21 27 16
7 10 23 20
3 25 15 20
1 11 10 29
2 2 6 14
6 28 15 3
3 30 21 3
5 18 1 12
3 14 12 15
5 20 18 19
6 14 6 26
0 20 20 4
5 22 24 26
6 31 17 31
5 15 4 5
3 25 5 9
2 5 5 9
4 31 8 17
3 13 9 18
5 7 14 28
8 5 23 15
2 29 5 10
5 3 9 29
2 7 27 3
2 21 0 4
5 30 21 0
3 23 21 13
1 4 9 30
4 0 17 4
2 27 6 22
2 9 25 9
6 28 21 27
5 18 12 21
3 26 13 26
4 13 9 2
3 22 12 23
4 19 11 12
3 4 16 29
4 4 20 0
7 10 17 27
8 26 21 1
2 27 10 0
7 8 10 17
8 11 29 18
8 29 29 24
0 19 24 8
4 27 10 0
7 5 7 25
7 20 13 25
4 1 19 3
3 20 7 7
3 2 4 21
0 12 27 15
2 2 15 1
8 27 19 13
2 5 11 21
8 16 6 1
1 28 22 28
3 18 7 6
6 19 2 19
8 12 5 15
5 8 17 27
1 30 31 4
1 15 3 2
7 4 13 7
5 31 15 18
1 15 2 23
3 20 15 22
8 25 14 13
3 3 27 2
0 10 5 11
7 22 12 21
4 17 1 4
2 3 8 5
3 0 27 31
3 9 5 21
7 8 15 6
7 20 8 0
7 0 23 18
2 9 5 13
3 0 1 7
1 16 4 11
3 15 17 27
6 5 28 31
0 1 12 29
3 20 28 30
7 31 1 28
4 31 23 0
8 11 14 16
1 12 20 2
6 20 0 19
6 20 25 17
7 17 13 31
0 11 24 0
6 31 13 4
4 8 17 29
7 1 5 11
