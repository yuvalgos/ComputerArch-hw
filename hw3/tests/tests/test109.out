Reading the opcodes latency info from tests/test109.dat ... Got latency for 9 opcodes
Reading the program file tests/test109.in ... Found 200 instructions
getProgDepth()==231
getInstDeps(13)=={-1,-1}
getInstDeps(107)=={20,40}
getDepDepth(10)==13
getInstDeps(106)=={104,70}
getInstDeps(93)=={40,33}
getInstDeps(171)=={113,137}
getDepDepth(52)==66
getDepDepth(25)==45
getDepDepth(169)==179
getDepDepth(171)==132
getDepDepth(163)==178
getInstDeps(199)=={186,196}
getDepDepth(67)==73
getDepDepth(10)==13
getInstDeps(52)=={40,44}
getDepDepth(40)==55
getDepDepth(120)==85
getDepDepth(18)==0
getInstDeps(102)=={83,88}
getDepDepth(95)==112
getInstDeps(45)=={37,-1}
getInstDeps(178)=={158,127}
getInstDeps(106)=={104,70}
getDepDepth(139)==101
getInstDeps(178)=={158,127}
getDepDepth(4)==0
getInstDeps(50)=={-1,42}
getInstDeps(40)=={22,28}
getInstDeps(110)=={68,97}
getInstDeps(133)=={118,86}
getInstDeps(61)=={45,42}
getDepDepth(55)==71
getDepDepth(119)==109
getInstDeps(104)=={92,82}
getInstDeps(27)=={-1,-1}
getDepDepth(11)==0
getDepDepth(43)==73
getInstDeps(36)=={34,17}
getInstDeps(125)=={86,124}
getInstDeps(172)=={157,156}
getInstDeps(198)=={195,147}
getDepDepth(30)==0
getInstDeps(133)=={118,86}
getDepDepth(195)==192
getDepDepth(148)==155
getInstDeps(82)=={24,73}
getInstDeps(89)=={54,44}
getDepDepth(27)==0
getDepDepth(83)==51
getDepDepth(75)==68
getInstDeps(30)=={-1,-1}
getDepDepth(186)==195
getDepDepth(155)==169
getDepDepth(0)==0
getInstDeps(167)=={137,126}
getDepDepth(32)==0
getDepDepth(170)==182
getDepDepth(74)==97
getDepDepth(64)==95
getDepDepth(17)==45
getInstDeps(117)=={98,71}
getDepDepth(104)==79
getInstDeps(82)=={24,73}
getInstDeps(186)=={160,168}
getDepDepth(10)==13
getInstDeps(92)=={62,43}
getDepDepth(123)==122
getInstDeps(96)=={56,55}
getDepDepth(144)==158
getInstDeps(96)=={56,55}
getInstDeps(146)=={124,135}
getInstDeps(142)=={80,132}
getDepDepth(123)==122
getInstDeps(7)=={-1,-1}
getInstDeps(171)=={113,137}
getDepDepth(192)==179
getInstDeps(95)=={-1,94}
getInstDeps(187)=={163,147}
getInstDeps(177)=={157,168}
getDepDepth(152)==163
getDepDepth(171)==132
getInstDeps(137)=={133,107}
getDepDepth(143)==131
getDepDepth(58)==74
getDepDepth(106)==98
getDepDepth(16)==11
getDepDepth(87)==18
getDepDepth(156)==174
getDepDepth(48)==45
getDepDepth(43)==73
getDepDepth(11)==0
getDepDepth(9)==0
getInstDeps(51)=={25,16}
getDepDepth(70)==34
getDepDepth(192)==179
getInstDeps(33)=={-1,-1}
getDepDepth(75)==68
getDepDepth(74)==97
getInstDeps(199)=={186,196}
getInstDeps(73)=={20,-1}
getInstDeps(162)=={156,148}
getDepDepth(47)==34
getDepDepth(149)==158
getInstDeps(13)=={-1,-1}
getDepDepth(21)==32
getInstDeps(78)=={45,61}
getDepDepth(195)==192
getInstDeps(0)=={-1,-1}
getDepDepth(132)==112
getDepDepth(30)==0
getDepDepth(106)==98
getDepDepth(45)==11
getDepDepth(139)==101
getDepDepth(22)==10
getInstDeps(95)=={-1,94}
getDepDepth(99)==54
getInstDeps(160)=={138,134}
getInstDeps(75)=={22,25}
getDepDepth(123)==122
getDepDepth(106)==98
getInstDeps(168)=={166,161}
getInstDeps(18)=={-1,-1}
getDepDepth(80)==68
getDepDepth(28)==50
getDepDepth(120)==85
getDepDepth(64)==95
getDepDepth(30)==0
getInstDeps(159)=={20,152}
getDepDepth(119)==109
getInstDeps(98)=={54,77}
getInstDeps(180)=={179,177}
getDepDepth(183)==174
getDepDepth(11)==0
getInstDeps(55)=={33,51}
getDepDepth(132)==112
getInstDeps(165)=={120,141}
getDepDepth(55)==71
getInstDeps(64)=={55,49}
getDepDepth(193)==196
